app/controllers/api/v1/GroupsController.js

```diff
@@ -17,6 +17,9 @@ exports.addController = function(app) {
       username: req.body.group.username,
       screenName: req.body.group.screenName
     })
+    if (req.body.group.hasOwnProperty('visibility')) {
+      newGroup.visibility = req.body.group.visibility
```

Comment:
Shall we filter invalid values in model defineProperty getter instead? My gut is saying this will slim the controller.